h/views/groups.py

```diff
@@ -49,15 +49,18 @@ def edit(self):
     def _js_config(self):
         csrf_token = get_csrf_token(self.request)
 
-        def api_config(route_name: str, method: str, **params):
+        def api_config(route_name: str, method: str, **kw):
             return {
                 "method": method,
-                "url": self.request.route_url(route_name, **params),
+                "url": self.request.route_url(route_name, **kw),
                 "headers": {"X-CSRF-Token": csrf_token},
             }
 
         js_config = {
             "styles": self.request.registry["assets_env"].urls("group_forms_css"),
+            "authenticatedUser": {
```

Comment:
`authenticatedUser` or just `user`?

Reply:
I've renamed this to `"user"` and also moved it into the `"context"` object alongside `js_config["context"]["group"]`